### PR TITLE
📝 Update roadmap - popup

### LINKED ISSUE #38: Remove popup from roadmap
Popup is implemented by https://github.com/zwin-project/zen/pull/346

### PR DESCRIPTION
**Pull request by Aki-7**
## Context

Popup is implemented in https://github.com/zwin-project/zen/pull/346

## Summary

- Mention that the popup was implemented.
  - Since it was difficult to explain the difference between popups and subsurfaces, I added postscript, instead of removing popup from roadmap.
- Add link to the twitter account.

### CODE REVIEW

File: content/roadmap/en.md [hunk 1]
```diff
@@ -10,6 +12,7 @@ XWayland runs on top of Wayland compositors and forward the windows from X11 app
 
 #### Popup / Subsurface
 Some wayland applications create another window on the main window. Right-click menu of Google Chrome is a good example. These are called Popup / Subsurface in Wayland, and Zen will support these features soon.
+*2022/01/23 - Popup has been implemented!*
```
Comment (rtr-dnd) [inline content/roadmap/en.md]: Let's add ✅ to the head of this line to emphasize this is done!

File: content/roadmap/ja.md [hunk 2]
```diff
@@ -10,6 +12,7 @@ XWaylandはWaylandコンポジッターの上で動作し、X11アプリケー
 
 #### Popup / Subsurface
 Waylandアプリケーションの中には、メインウィンドウ上に新たに小さいウィンドウを作るものがあります。Google Chrome上で右クリックを行ったときに出現するメニューなどはその一例です。Popup / Subsurface などと呼ばれるこれらの機能についても、Zen上での近日中のサポートを予定しています。
+*2022/01/23 - Popupについては実装されました！*
```
Comment (rtr-dnd) [inline content/roadmap/ja.md]: I suggest changing this line to「✅ 2022/01/23 - Popupを実装しました！」

**Comment by rtr-dnd** [comment]
Just noticed 3D app development page (/getting_started/3d_app_development) also has some part which needs updating. 
I recommend searching for "popup" through the entire repository.

File: content/getting_started/3d_app_development/en.md [hunk 3]
```diff
@@ -14,9 +14,6 @@ The Beta version of Zwin is still incomplete and has the following caveats relat
   - In particular, apps such as VSCode will not work.
   - Instead, **use a CUI editor such as Vim or nano**.
 
-- Some functions of the Wayland client do not work.
-  - **Pop-up windows do not open when right-clicking, etc.**
-
```
Comment (Aki-7) [inline content/getting_started/3d_app_development/en.md:19]: Some functions of the Wayland client (such as relative pointer) still do not work, but they are difficult to describe and don't seem to matter so much, so I just removed these lines.

**Comment by rtr-dnd** [comment]
LGTM🫡

**Pull request merged**